TM1py/Services/HierarchyService.py

```diff
@@ -428,6 +429,7 @@ def update_or_create_hierarchy_from_dataframe(
             verify_edges: bool = True,
             element_type_column: str = 'ElementType',
             unwind: bool = False,
+            unwind_consol: list = [],
```

Comment:
We shouldn't use a mutable default value, use `None` instead and handle the argument accordingly.